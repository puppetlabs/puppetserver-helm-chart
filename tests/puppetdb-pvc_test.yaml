suite: test puppet puppet volume creation
templates:
  - puppetdb-pvc.yaml
release:
  name: puppetserver
  namespace: puppet
tests:
  - it: should not create the volume if "puppetdb.enabled" is false
    set:
      puppetdb.enabled: false
    asserts:
      - hasDocuments:
          count: 0
  - it: should create the volume by default
    asserts:
      - isKind:
          of: PersistentVolumeClaim
