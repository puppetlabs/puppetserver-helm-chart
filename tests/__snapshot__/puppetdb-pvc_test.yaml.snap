manifest should match snapshot:
  1: |
    apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      annotations: null
      labels:
        app.kubernetes.io/component: puppetdb
        app.kubernetes.io/instance: puppetserver
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: puppetserver
        app.kubernetes.io/version: 7.17.0
        helm.sh/chart: puppetserver-9.4.0
      name: puppetserver-puppetdb-claim
    spec:
      accessModes:
        - ReadWriteOnce
      resources:
        requests:
          storage: 400Mi
